docs/RULES.md

```diff
@@ -1087,3 +1088,28 @@ To fix this, used fenced code blocks:
     Some more text.
 
 The reverse is true if the rule is configured to use the `indented` style.
+
+## MD047 - File should end with a blank line
+
+Tags: blank_lines
+
+Aliases: file-should-end-with-blank-line
+
+This rule is triggered if the last line of the file is not empty.
```

Comment:
That's not entirely accurate, it's more like "If the file does not end in a newline"